mteb/abstasks/AbsTask.py

```diff
@@ -32,6 +34,31 @@ def dataset_transform(self):
         """
         pass
 
+    def stratified_subsampling(
+        self,
+        splits: Union[str, list[str]] = ["test"],
```

Comment:
I would just use list[str] (following the mantra: `There should be one-- and preferably only one --obvious way to do it.`)